src/components/WebCLI/_webcli.scss

```diff
@@ -0,0 +1,90 @@
+@import "../../scss/settings";
+@import "vanilla-framework/scss/vanilla";
+
+.webcli {
+  bottom: 0;
+  position: fixed;
+  width: calc(100vw - #{$sidebar-collapsed});
+
+  &__input {
+    background-color: black;
+    display: flex;
+    height: 2.5rem;
+    padding: 0 1rem;
+
+    &-prompt {
+      color: #39b54a;
+      cursor: default;
+      flex-grow: 0;
+      // https://github.com/canonical-web-and-design/vanilla-framework/pull/3370
+      // XXX The unquote will need to be removed with the next vanilla release
+      font-family: unquote($font-monospace);
+      font-weight: bold;
+      line-height: 2.5rem;
+      padding-right: 0.5rem;
+    }
+
+    form {
+      flex-grow: 1;
+    }
+
+    &-input[type="text"] {
+      background-color: black;
+      color: $color-mid-light;
+      // https://github.com/canonical-web-and-design/vanilla-framework/pull/3370
+      // XXX The unquote will need to be removed with the next vanilla release
+      font-family: unquote($font-monospace);
+      height: 2.5rem;
+      margin-bottom: 0;
+      padding: 0 0.5rem;
+    }
+
+    &-help {
+      flex-grow: 0;
+      line-height: 2.5rem;
+      margin-left: 1rem;
+
+      i {
+        cursor: pointer;
+      }
+    }
+  }
+
+  &__output {
+    background-color: #2c001e;
```

Comment:
Would be good to have a nicely named variable for this as a setting at the top of this file

Reply:
done